packages/deployer/test/fixture-projects/sample-project/contracts/modules/InitializableModule.sol

```diff
@@ -0,0 +1,19 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "../storage/InitializableStorage.sol";
+import "@synthetixio/core-contracts/contracts/initializable/InitializableMixin.sol";
```

Comment:
Super nitpick, but I like to sort imports by "distance", so it would be like this:
```suggestion
import "@synthetixio/core-contracts/contracts/initializable/InitializableMixin.sol";
import "../storage/InitializableStorage.sol";
```